test/Microsoft.AspNet.Mvc.TagHelpers.Test/AnchorTagHelperTest.cs

```diff
@@ -124,13 +129,15 @@ public class AnchorTagHelperTest
                 allAttributes: new Dictionary<string, object>(),
                 items: new Dictionary<object, object>(),
                 uniqueId: "test",
-                getChildContentAsync: () => Task.FromResult("Something"));
+                getChildContentAsync: () => {
+                    var tagHelperContent = new DefaultTagHelperContent();
+                    tagHelperContent.Append("Something");
+                    return Task.FromResult((TagHelperContent)tagHelperContent);
```

Comment:
does compiler seem to require a cast here?  if not, leave it out.  if yes, use `Task.FromResult<TagHelperContent>(tagHelperContent)` instead.